docs/references/expo/use-local-credentials.mdx

```diff
@@ -38,29 +38,28 @@ The `useLocalCredentials()` allows you to store a user's password credentials on
   - `setCredentials()`
   - <code>(params: [LocalCredentials](#local-credentials)) => Promise\<void></code>
 
-  Stores the provided credentials on the device if the device has enrolled biometrics. The end user needs to have a passcode set in order for the credentials to be stored, and those credentials will be removed if the passcode gets removed. The promise will reject if value cannot be stored on the device.
+  Stores the provided credentials on the device if the device has enrolled biometrics. The end user needs to have a passcode set in order for the credentials to be stored, and those credentials will be removed if the passcode gets removed. The promise will reject if the value cannot be stored on the device.
 
   ---
 
   - `clearCredentials()`
   - <code>() => Promise\<void></code>
 
-  Removes the stored credentials from the device. The promise will reject if value cannot be deleted from the device.
+  Removes the stored credentials from the device. The promise will reject if the value cannot be deleted from the device.
 
   ---
 
   - `authenticate()`
-  - <code>() => Promise\<SignInResource></code>
+  - <code>() => Promise\<[SignInResource](/docs/references/javascript/sign-in/sign-in)></code>
 
-  Attempts to read the stored credentials and creates a sign in attempt with the password strategy. A promise with a SignInResource if the stored credentials were accessed, otherwise the promise will reject.
+  Attempts to read the stored credentials and creates a sign-in attempt with the password strategy.The promise will resolve to a `SignInResource` if the stored credentials were accessed. Otherwise, the promise will reject.
 </Properties>
 
-
 ### `LocalCredentials`
 
 <Properties>
   - `identifier`
-  - `?string`
+  - `string`
```

Comment:
@alexisintech I did this on purpose, because this property is optional, is there any other way to highlight this ?

Reply:
then the `?` goes after - like with typescript

Reply:
let me change that :)